README.md

```diff
@@ -64,6 +64,7 @@ No. Right now you cannot.
 * Improved - Improved code with phpcs.
 * Removed  - Twitter commented code.
 * Updated  - Plain texts to WPi18n functions.
+* Compatibility testing with 5.6.
```

Comment:
@juhi123 

`Compatibility testing with 5.6.
Write` 

This is in two release changelog. Please remove from one.